include/criterion/options.h

```diff
@@ -116,6 +116,14 @@ struct criterion_options {
      *  default: false
      */
     bool wait_for_clients;
+
+    /**
+     * Raise a signal to crash the test if a assert fails so that an debugger
```

Comment:
This should be "if an assert" and "that a debugger".

Also, this no longer raise a signal -- this could be reworded as "Raise a debug trap"